tests/testthat/test_plot_grid.R

```diff
@@ -27,6 +27,10 @@ test_that("basic plot arranging works", {
   expect_doppelganger("scaling plots",
     plot_grid(p1, p2, p3, labels = "AUTO", scale = c(.9, .7, .5), nrow = 1) + theme_map() # add theme_map() for plot title
   )
+
+  expect_doppelganger("colwise arranging",
+    plot_grid(p1, NULL, p2, NULL, p3, NULL, byrow = FALSE) + theme_map()
```

Comment:
I'd delete the last `NULL` argument to test the case that the number of plots is less than `rows * cols`.

Reply:
Thanks, yes, of course this could break things. I'll check and fix it: adding a visual test that assesses the case where the number of plots is less than `rows * cols`